nixos/modules/services/networking/tetrd.nix

```diff
@@ -0,0 +1,28 @@
+{ config, lib, pkgs, ... }:
+
+{
+  options.services.tetrd.enable = lib.mkEnableOption pkgs.tetrd.meta.description;
+
+  config = lib.mkIf config.services.tetrd.enable {
+    environment.systemPackages = with pkgs; [ tetrd nettools ];
+
+    systemd = {
+      # Seems to need root access for /run/tetrd.sock, which seems to be hard-coded
+      services.tetrd = {
+        description = pkgs.tetrd.meta.description;
+        wantedBy = [ "multi-user.target" ];
+
+        serviceConfig = {
+          ExecStart = "${pkgs.tetrd}/opt/Tetrd/bin/tetrd";
+          Restart = "always";
+        };
+      };
+
+      # Unfortunately, it appears these paths are hard-coded in
+      tmpfiles.rules = [
+        "L+ /usr/bin/route - - - - ${pkgs.nettools}/bin/route"
+        "L+ /usr/bin/ifconfig - - - - ${pkgs.nettools}/bin/ifconfig"
+      ];
```

Comment:
Maybe this software belongs in [NUR](https://github.com/nix-community/NUR) instead of mainline?

Reply:
Is there any nix magic we can do to get around the seemingly hard-coded paths?

Reply:
Hmm... I have [seen](https://github.com/NixOS/nixpkgs/blob/a640d8394f34714578f3e6335fc767d0755d78f9/nixos/modules/services/networking/unifi.nix#L175) some `systemd` hackery with [BindPaths](https://www.freedesktop.org/software/systemd/man/systemd.exec.html#BindPaths=) before. Care to give that a try?